adding a blog post about GAAD (#110)
ember-learn/ember-blog: source/2019-05-13-global-accessibility-awareness-day.md

## 5-30 minutes

- Visibly support accessibility efforts on your social media platform of choice. Here are some ideas: 
  - Share the link to GAAD, and vocalize that all websites should be accessible. Make sure to include the hashtag A11y (`#A11y`).
  - Start following (if you aren't already) prominent Accessibility Advocates & Engineers. Programmers like [Léonie Watson](https://twitter.com/LeonieWatson), [Scott O'Hara](https://twitter.com/scottohara), [Marcy Sutton](https://twitter.com/marcysutton), [Eric Bailey](https://twitter.com/ericwbailey), [Carie Fisher](https://twitter.com/cariefisher), and [Zoë Bijl](https://twitter.com/ZoeBijl) are all a great place to start!
  - Share your own advice on how to get started with digital accessibility. 
- Find the existing keyboard shortcuts for your operating system, and schedule time to become more familiar with them, via https://dequeuniversity.com/screenreaders/. 

## 30-60 minutes

> - Go mouseless: using only your keyboard, visit a website that you helped build (even if it's just a local build) and try to navigate around the app with only your keyboard. Don't have a website to visit? Visit our own website (emberjs.com) and try it out- and if you find things that don't work, file an issue or submit a PR!

Reviewer: I think it'd be

```suggestion
- Go mouseless: using only your keyboard, visit a website that you helped build (even if it's just a local build) and try to navigate around the app with only your keyboard. Don't have a website to visit? Visit our own website (emberjs.com) and try it out - and if you find things that don't work, file an issue or submit a PR!
```

- Check the modals/dialogs/popups in your application. Do they trap focus? File an issue or submit a PR if they do not. Not sure what that means? Check out [this simple modal addon](https://github.com/melsumner/e-a11y-modal) for a demonstration. 

## 60+ minutes

- Are you ready to walk a mile in another person's shoes? [Watch a user with a screen reader navigate the web](https://www.smashingmagazine.com/2019/02/accessibility-webinar/)
- Read [“Semantics to Screen Readers”](https://alistapart.com/article/semantics-to-screen-readers) and learn how the code we write influences what someone with a screen reader gets from our applications. 

> - Look for an un-accessible code samples in the Ember.js guides, and submit a PR to fix one. Or two. Or ten.

Reviewer: I think it'd be

```suggestion
- Look for an un-accessible code sample in the Ember.js guides, and submit a PR to fix it. Or two. Or ten.
```

> As for me, I'll be spending May 16th in the `#topic-a11y` channel on the [Ember Community Discord](https://discord.gg/emberjs) chat server- ready to help you with whatever accessibility questions you might have. If you are able to help answer questions and knowledge share that day as well, please do join in!

Reviewer: I think it'd be

```suggestion
As for me, I'll be spending May 16th in the `#topic-a11y` channel on the [Ember Community Discord](https://discord.gg/emberjs) chat server - ready to help you with whatever accessibility questions you might have. If you are able to help answer questions and knowledge share that day as well, please do join in!
```
